subcommand/mesh-init/command.go

```diff
@@ -63,31 +60,68 @@ func (c *Command) Run(args []string) int {
 }
 
 func (c *Command) realRun() error {
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
 	taskMeta, err := awsutil.ECSTaskMetadata()
 	if err != nil {
 		return err
 	}
 
 	cfg := api.DefaultConfig()
 
+	consulClient, err := api.NewClient(cfg)
```

Comment:
Keep the old version of the client intact so that unit tests don't fail in this branch. WIll eventually be removed when we start registering services with the new version of the client

Reply:
nit: Maybe add a `TODO` comment to keep track of this